eclipse-sirius/sirius-components
# [1248] Add Support for PieChart in the Form representation

This PR adds the D3 Pie Chart in the Form representation.
Solve #1248 

# Pull request template

## General purpose
What is the main goal of this pull request?

## Project management

## Architectural decision records (ADR)

## Dependencies

## Frontend

This section is not relevant if your contribution does not come with changes to the frontend.

### General purpose

### Typing
We need to improve the typing of our code, as such, we require every contribution to come with proper TypeScript typing for both changes contributing new files and those modifying existing files.
Please ensure that the following statements are true for each file created or modified (this may require you to improve code outside of your contribution).

- Hooks are properly typed:

## Backend

This section is not relevant if your contribution does not come with changes to the backend.

### General purpose

### Architecture

## Review

### How to test this PR?
Import the given View model: [View.zip](https://github.com/eclipse-sirius/sirius-components/files/8871000/View.zip)
It contains a Form Description with a BarChart and a PieChart descriptions.
Create a new Robot Flow model
Create a new Form Description on System.
It should display the two charts representing the volumes of processors.
